man/adoc/bpftrace.adoc

```diff
@@ -2350,6 +2358,28 @@ tracepoint:syscalls:sys_enter_execve {
 55f683ee2000-55f683ee3000 rw-p 00000000 00:00 0
 ----
 
+[#functions-pid]
+=== pid
+.variants
+* `uint32 pid([PidNamespaceMode mode])`
```

Comment:
ah sorry, this should be updated as well as `PidNamespaceMode` no longer exists.

Reply:
Yes, I was going to ask what I should change it to.

Reply:
Yeah, that's what I thought as well.